src/pages/Staking/TotalStakedUI/index.tsx

```diff
@@ -0,0 +1,67 @@
+
+import { useQuery } from 'react-query';
+import {
+  useErrorHandler,
+  withErrorBoundary
+} from 'react-error-boundary';
+import { useSelector } from 'react-redux';
+
+import InformationUI from '../InformationUI';
+import ErrorFallback from 'components/ErrorFallback';
+import {
+  VOTE_GOVERNANCE_TOKEN_SYMBOL,
+  VoteGovernanceTokenMonetaryAmount
+} from 'config/relay-chains';
+import { displayMonetaryAmount } from 'common/utils/utils';
+import genericFetcher, { GENERIC_FETCHER } from 'services/fetchers/generic-fetcher';
+import { StoreType } from 'common/types/util.types';
+
+const TotalStakedUI = (): JSX.Element => {
+  const { bridgeLoaded } = useSelector((state: StoreType) => state.general);
+
+  const {
+    isIdle: totalVotingSupplyIdle,
+    isLoading: totalVotingSupplyLoading,
+    data: totalVotingSupply,
+    error: totalVotingSupplyError
+  } = useQuery<VoteGovernanceTokenMonetaryAmount, Error>(
+    [
+      GENERIC_FETCHER,
+      'escrow',
+      'totalVotingSupply'
+    ],
+    genericFetcher<VoteGovernanceTokenMonetaryAmount>(),
+    {
+      enabled: !!bridgeLoaded
+    }
+  );
+  useErrorHandler(totalVotingSupplyError);
+
+  let totalStakedVoteGovernanceTokenAmountLabel;
+  if (
+    totalVotingSupplyIdle ||
+    totalVotingSupplyLoading
+  ) {
+    totalStakedVoteGovernanceTokenAmountLabel = '-';
+  } else {
+    if (totalVotingSupply === undefined) {
+      throw new Error('Something went wrong!');
+    }
+    totalStakedVoteGovernanceTokenAmountLabel = displayMonetaryAmount(totalVotingSupply);
+  }
+
+  return (
+    <div>
+      <InformationUI
+        label={`Total Staked ${VOTE_GOVERNANCE_TOKEN_SYMBOL}`}
+        value={`${totalStakedVoteGovernanceTokenAmountLabel} ${VOTE_GOVERNANCE_TOKEN_SYMBOL}`} />
+    </div>
```

Comment:
For now, only the total staked vKINT is displayed because the API for the total staked KINT is not ready yet. But once @savudani8 exposed that API, I can add it too.

Reply:
For now, I'll resolve this.

Reply:
For now, I will resolve this.